### PR TITLE
Update blocklist_29.02.2024

### PR DESCRIPTION
**Pull request by matej402**
https://xray.helius.xyz/token/3KgygnhgPy7ZNy2jYmVzj2dX1P2dGYB7TgW9bXyneJHb?network=mainnet https://xray.helius.xyz/token/2VuS1MmddudXbGwnpYbhGjKyKDRTsNR1GCBb4WCppwRN?network=mainnet https://xray.helius.xyz/token/737W8meHTQcTpwyZU6usBJpJnuSD6fmSkupn7XKzXAr1?network=mainnet https://xray.helius.xyz/token/Enio2vZikpBsgbhcmNUAnpD7rdRi3rNURcW3b6ei5S7R?network=mainnet https://xray.helius.xyz/token/3ByfZbwEy2hFacVo48qfG9JUSamsi9EwnMrNuRvpXgpc?network=mainnet https://xray.helius.xyz/token/FdhLc4e4Ce5zVzaK1C641S3NmkGPZnGDLSEpAU275jZA?network=mainnet https://xray.helius.xyz/token/HBWwECEAovWmSjAAWpKNTnyF5bWHSK7EgMrhiYf8qc9d?network=mainnet https://xray.helius.xyz/token/2MjRXLLuipdQSmr1tPEBQD2TVZELLs5eUgu4wWMCTwK3?network=mainnet https://xray.helius.xyz/token/BNJVX8jpjsVWj7xutJcktrLQTCSKSXHfaZoAyzzG9pxK?network=mainnet https://solscan.io/token/8aNjS2QVN7mUoihS4NzDxsAFjee6nQyQvfJKdNnhejQh#metadata https://xray.helius.xyz/token/BeELS5MC4mtELGinyGav3CVyS9BMM5P6wmTVvqJ5ZtG9?network=mainnet https://solscan.io/token/BeELS5MC4mtELGinyGav3CVyS9BMM5P6wmTVvqJ5ZtG9#metadata https://solscan.io/token/GpQ2LHHVgD8DTpDTjhb77RHBEPWn5D5TukQFJvZ9ZJAc https://solscan.io/token/CoNf7uAR762EmVzEa96zgKMrK4QAAgKy6EXhhJbr6u3i https://xray.helius.xyz/token/BrV2Ri2tZajUQhVewXfUtmyab9XPgoj51HFpEW2Jknji?network=mainnet https://xray.helius.xyz/token/GosCDtGbRuqshhdPrnAaNFYeKJ7MLXHeFLvWCxrpeU8Q?network=mainnet https://xray.helius.xyz/token/G6qVPgYMQjsgZb4697Y2Gs7A94QQkj2Li6mpkCNUgrhQ?network=mainnet https://xray.helius.xyz/token/JAYVJ23mKRaSvxHDfij6Lqu6fjGPEpfP2HS8yV4zRfF8?network=mainnet https://xray.helius.xyz/token/J2BS7q98pdeXq6DufEJ4CLhE1v4BhmKVTn7mC5HSKiNv?network=mainnet https://xray.helius.xyz/token/9t46mr9qditjXjfquBYh262FXxyEfmrfP4BR4SX9QYL1?network=mainnet https://xray.helius.xyz/token/EafE5sNQWJ2FiKsB8EWR7rjsTPBUBRQ7xDxsVmp2HWLU?network=mainnet https://solscan.io/token/4PTtEhifKpf8T4Z9ac9zLeQX3xuuTWCdEsjAgYYSHSei#metadata https://solscan.io/token/4NcdTKTcfYvv5Vb4s4vHa3EqVtRvPkQHaYSjfgcmxEHP#metadata https://xray.helius.xyz/token/CyJxbTf5dajtVwPwrFTa2sij15nahzeMup9vnXMooDzp?network=mainnet https://solscan.io/token/KPetYEd6neCkUcKcbBeiLBesBGc5iVnoDK8sKv9w3Up https://solscan.io/token/BraZ66yWmrreegJV35nLZnniNE6MH5R3iQMs789uCJuJ#metadata https://solscan.io/token/2zxHnSq6dD1Jsr8kScd74dkhN6PZ1JwUQM6LeszpaMD7 https://solscan.io/token/51DAbnU2n3TuaBGHCfYw4Gy2t1jzMjMpbCFo1FKejr7r#metadata https://solscan.io/token/Gw1iwVMAeeg2R7GAuWF4AigU3hQEycDzMju12xSJkSgx#metadata https://solscan.io/token/AzBV8BqCXGtZc3Agye2EjucsHBkYEetwjczwburDWMi9 https://solscan.io/token/3sfqsAbDYaTYAVGYxuRR7s2wGMoyKkeMJUdG7HfMC8Zy#metadata https://solscan.io/token/BDsgjSWE93tSDnw3sT9SWj1y5M9DUaFeTedBCzmsuUdj#metadata https://solscan.io/token/Az6VWvGV19G8Jn359cXEBU7kCKep8gkLgyeaGLCqedns#metadata https://xray.helius.xyz/token/7ribUw5yEAbREt9stcrNMC5LMw71HK6WoBoLQdmS2q1s?network=mainnet https://xray.helius.xyz/token/4JkFzotCQhwC9iCKpMpWcmJ7awb7q98oVmWZDW7X53R7?network=mainnet https://xray.helius.xyz/token/nNuqHfgT4aVo6jTSVw2PUxdosy5tk5ue4Ba69mm6rD6?network=mainnet https://xray.helius.xyz/token/DXVBWTVQxs4bP1XXcXfztrvRnHQ69uSUyhuUvFRL6cWF?network=mainnet https://solscan.io/token/HxRJvijSNFrC3vNJdZwitSbrgfftiuzPPvMcHPTyh5T9 https://solscan.io/token/2J3tabmsVmqz6bbETGBNJarV72DuUrYQWcemU9Hby9t2#metadata https://solscan.io/token/BCjd3v11DXSv7bKrDkVrgrqNE9N7bGbeTwtTn7L1h62X#metadata https://solscan.io/token/4jEEBk1cWg4vEgR8ULMVEFBt9AaELNPDKfS9WfVwAWrM https://solscan.io/token/7HSpjyQR6mxk7WxycsD7KZCZHtxVuE4upSMyWErR1Eyj https://solscan.io/token/659W98ePoKDJ4bwLzDFCTqSbfxYgbpf2BU9reCuRWhdz#metadata https://solscan.io/token/EMP58FcZR4iA4uJRaL1R88qhEPYZAeoVF5KFHpyVEVh2#metadata https://solscan.io/token/8AF9jyyL3DkAh6bhTm18AYVrC7DTH8NCpsL5dCqjB1Jb#metadata https://xray.helius.xyz/token/J1WRCizr3e36SXSzK2HyFoB8mCiA56M1WCQ2mhLpTGzr?network=mainnet https://solscan.io/token/6JRwwCxMxsfXoQnMJmSKtaKdnEtLQD44HZvhoyJC5xSc#metadata https://solscan.io/token/CJXP5DbQiej89bBor5pRnJun3PgpmvKU7bWMZPyUueAC#metadata https://solscan.io/token/4yrMHaPCTbasocHJjymJC5qB5dNjeHdsLvxonVX27gqF https://solscan.io/token/8NVzLVthJh3qNzy4LiWsAR4WSY64iwnSvh8mTfKr9jnY https://solscan.io/token/FAwMgTfFiJqiBgqyWUx2YXDMeTeKVzU45ck3JuiEQYj1#metadata https://solscan.io/token/3iFn4yDbjDjwU2pK8L6t3NydK8Kaev2aBUs3xxyRtQB4#metadata https://solscan.io/token/GvLFXp5DqowBtF4nUjXmqx6upH2bjncf2ESndH6ZF1sA#metadata https://solscan.io/token/EwTS9YRE6oxvnKUyNz4wN4YydBWV5CAFXNqM84ckSPS4#metadata https://xray.helius.xyz/token/4zi8LuhB6GfdFFri4YaHoLUKdTtZSfRNHct6kAD7s34D?network=mainnet https://solscan.io/token/7P5WKj3AxF22EGuqt1rFrxuoi2Z5aABsqvxaf6R6YyrP#metadata https://solscan.io/token/6y5SW5xYmqHcwe9jmaqA441Lzn9SL8oyeHn94Sy1TgHn#metadata https://solscan.io/token/2HjHFqBPTnHccp4k8sfea4cWdu1pxms65fBYQASPnjbr https://xray.helius.xyz/token/Hc9LQCuGfrpoxiQQab81ig9hbVVfE5XvR5u4sKmSH4oK?network=mainnet https://xray.helius.xyz/token/GUBoApWqD65Vzw4hzRrLcjicLV2j9wftE4GhsQqLKzmh?network=mainnet https://xray.helius.xyz/token/3rv8oRHN1S9dygbtNR71QQQkc7MozAnxgKmSYJuYJPYe?network=mainnet https://xray.helius.xyz/token/BqoNJfyNscD8a7X7iW6Jiknr349t9rb68EhR52tddCjb?network=mainnet https://xray.helius.xyz/token/9ZYqzZmveaAnRLVEBD7uX9iKwqvzrGy9UXYngvwoVsQW?network=mainnet https://xray.helius.xyz/token/DHakKM338T6KJRQwJ3dpb626vAeNLpnvnj3bGVuqXCXz?network=mainnet https://xray.helius.xyz/token/3rv8oRHN1S9dygbtNR71QQQkc7MozAnxgKmSYJuYJPYe?network=mainnet https://xray.helius.xyz/token/k5MHkwHeJpK46i1S5LmnbTrh2BhQpVFud2yXqHMooYC?network=mainnet https://xray.helius.xyz/token/BMXdyYJYQTmgf9tGhGqytk4pjb5XQUxYL3jJWnzLyKER?network=mainnet https://solscan.io/token/3qsteoA9xaXAfsY5XLA85TiUg1nCWigfZUsRoGNtH4tj#metadata https://xray.helius.xyz/token/5xxMsYuK5ZkMf3dgyc6BH4D8QkqnD3TrFKwWpgXjiQt1?network=mainnet https://solscan.io/token/7FxSs1p9A5GyjMFmBZDKooxiPHwu2yZA6NL3E1dqX8sx#metadata https://solscan.io/token/EKaY1wKkXQq88NcRP9Gs3dazAdVjXbPtYgubsBjXcHJg#metadata https://xray.helius.xyz/token/5KE5GftGJ7G9w8so1Wt5FNjM1kGiBotab8Xf8E2xz25Q?network=mainnet https://solscan.io/token/4YgEpLidsZEEpkCDFyshq68mtZGZr66Sf4SZf4qRPGwd https://xray.helius.xyz/token/JBqzQhKVb1FL5Jq9RuZ7anSAGKj7oY7BTUC7pBCDUTFF?network=mainnet https://xray.helius.xyz/token/PN1gkQBNUwj6RZqSnUpMdBz2KdhdgWog1MvPRbFuXEB?network=mainnet https://xray.helius.xyz/token/JBqzQhKVb1FL5Jq9RuZ7anSAGKj7oY7BTUC7pBCDUTFF?network=mainnet https://xray.helius.xyz/token/DYN4YoxEYiRKwomnqRWnLAHtfsrfQskqcpMvfz29QJKL?network=mainnet https://xray.helius.xyz/token/5pivz2m2yXk9kXNJb3cL4TRsexNf1rY89nGVB6Bcegwy?network=mainnet https://solscan.io/token/5S54pc315qY8Cp1PzXtps7dwfFmY1RhYZnqrebpe5P7g https://solscan.io/token/4zHSFQoVfUYUCNnExcEif1wed4w1aYzQHRTobbVfsamT https://solscan.io/token/9mbrbDne1qs49175Eia2zccUUN8pjkipJ2teY6o8zC1u https://xray.helius.xyz/token/AKAVEfjswg2xruVwXJqn8iDMiBdzb53apLPAoSVLJGoE?network=mainnet https://solscan.io/token/3t4j74eR1xS9preLgtqTSUMp2RJ21eD7MQ2GUHvRK5G5 https://solscan.io/token/GHY8WyVxWzSpLTs85jvohUz6uyP3RMCV3n5oJNR6cULq#metadata https://solscan.io/token/4ss7nNUfXVXedqmpm1BUhjZyf677DAfK65yNurRxhxWb https://xray.helius.xyz/token/FWnpgUhg2mtonkRZupYG7GSCeJUMjNdk32DiiFqTjH2J?network=mainnet https://xray.helius.xyz/token/GT93DUvQskCV8gTTdEqjrnG9oPXWntwsvgZhBL2oyj99?network=mainnet https://xray.helius.xyz/token/5SLZQFYpEx6EGMYyAucBs593yRp2VjTqyceSx1BPRNA5?network=mainnet https://solscan.io/token/Ch9K4fNJ9iiPbqF9cWiePnVMgX3LzqPo7xefCJMCFRbT#metadata https://solscan.io/token/82omU6ya8h98UxFyRbDyzm2n3V6VTtMTnJqkcSv7N2YV https://xray.helius.xyz/token/H31siuoaeByMqEE8etwypyge3QoxQoNVRLYaKAMVruKK?network=mainnet https://solscan.io/token/8LpoGA5176sGviYQskefC6j19ww5sMGfiV69xL43UERy https://solscan.io/token/Gex35fmz4docKVXjkRbZ6ZJvtnr3ybytG74iaGNeNqvg#metadata https://solscan.io/token/Gex35fmz4docKVXjkRbZ6ZJvtnr3ybytG74iaGNeNqvg https://solscan.io/token/8djbRAQpGKqJNRdBCaTCXgBK2C3Yo9Ssi7DTTffU6Kk8 https://xray.helius.xyz/token/3P9uBQdGAyRTbeCfaR7sodjMWyBoq9rmMZvVPRKLe8VP?network=mainnet https://solscan.io/token/7WiJLr6MEJoTBfsNpwcQALXX7gz6bVX9iZejHrm6uEdt https://xray.helius.xyz/token/7yepA1ZY6GQBcdgLSg2HYBDqPRm8q8PgGborN6Yo7DjV?network=mainnet https://solscan.io/token/GzMdi8W6m5E4WFn6aycN6yeY4ARDLn16kN33iwyAYHYG https://solscan.io/token/E7aRjjqUr4TdsU2vDjg5KhCUivCx394JwvmeEidjAp7U https://solscan.io/token/33Tae1N4tbS6YBNxa371GvZpb8PPpeV3p4vv76xiCpc3 https://solscan.io/token/9se94v3wVJsHSaheRrgaofPudXvCwPTa9yyjhP3HfCFK https://xray.helius.xyz/token/2eq56zoJ6ZfxbZBx7q6A93mX6YHRKcFcsyG6zhrWvYgS?network=mainnet https://solscan.io/token/9zjuVgYJmztoDvz2CtGJezei9hoaUY5wC8hKHDQheydr https://solscan.io/token/9XvJ2xFnqWRPtxzboh6v2poYxbwrhy5rRqtJckgzgQE1 https://solscan.io/token/FSryK8wP6g5psaKwukA7gg6LWvZnXVCkGPcGTq2euPWg https://solscan.io/token/HbpiVyN2AAK6nffhcZp6MRWLEmV4o1xoCYxZ7zVSJWDC

### CODE REVIEW

File: lists/blocklist.yaml [hunk 1]
```diff
@@ -44,3 +44,63 @@
...
+  - url: jupiter-drop.netlify.app
+  - url: meme-draw.io
+  - url: solanbox.com
+  - url: soldrop.online
+  - url: pyth.bio
+  - url: dropray.site/
+  - url: rollbitking.com
+  - url: sol2024.info
+  - url: nftarea.net/
+  - url: wenclaim.net
```
Comment (skakac) [inline lists/blocklist.yaml:103]: Remove "/" from urls like "nftarea.net/" and others

File: lists/blocklist.yaml [hunk 2]
```diff
@@ -44,3 +44,63 @@
...
+  - url: staratlas.bio
+  - url: jupiter-drop.netlify.app
+  - url: meme-draw.io
+  - url: solanbox.com
+  - url: soldrop.online
+  - url: pyth.bio
+  - url: dropray.site/
+  - url: rollbitking.com
+  - url: sol2024.info
+  - url: nftarea.net/
```
Comment (filipljusic) [inline lists/blocklist.yaml:102]: remove `/`

**Review by skakac — CHANGES_REQUESTED** [review]

File: lists/blocklist.yaml (hunk 1 above)
Comment (matej402) [reply lists/blocklist.yaml:103]: removed, thank you

File: lists/blocklist.yaml (hunk 2 above)
Comment (matej402) [reply lists/blocklist.yaml:102]: removed, thank you

**Pull request merged**